### PR TITLE
fix(app page): fix phone validation

### PR DESCRIPTION
**Pull request by lavanya-bmw**
## Description

Fixed validation error for phone number field

## Why

To fix validation error

## Issue

Ref: TEST-1166

## Checklist

Please delete options that are not relevant.

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally

### CODE REVIEW

**Comment by manojava-gk** [comment]
Kindly fix the Sonar Bug

File: src/types/Patterns.ts [hunk 1]
```diff
@@ -73,7 +73,7 @@ export const Patterns = {
       /^[^ @=<>*\-+#?%&_:;]([a-zA-Z0-9 !?@&#'"()[\]_\-+=<>/*.,;:%\r\n]){9,1999}$/,
     longDescriptionDE:
       /^[^ @=<>*\-+#?%&_:;]([a-zA-ZÀ-ÿ0-9 !?@&#'"()[\]_\-+=<>/*.,;:%\r\n]){9,1999}$/,
-    phone: /^\+(\d{2})+(\(\s\d{3}\))?\s?\d{9,20}$/,
+    phone: /^\+(\d{2})?(\s?)+(\(\d{3}\))?\s?\d{10,13}$/,
```
Comment (oyo) [inline src/types/Patterns.ts:76]: Three things here:
1. fix the sonar bug - something like changing `(\s?)+` to `\s+` should work
2. add a convenience method `isPhone` for that. Scroll down to line 90+ and add it similar to the existing methods.
3. regular expressions are among the things where unit tests make the most sense. Especially if they are not self explanatory like this one here. Someone before added this regex and forgot to add tests. Go to `Patterns.test.ts` and add values for valid and invalid phone numbers similar to the existing tests. Then run `yarn test Patterns` to check if to regular expression works correctly.

**Review by oyo — CHANGES_REQUESTED** [review]

**Comment by jjeroch** [comment]
decision: pr against dev instead of 1.6.0

**Review by oyo — APPROVED** [review]

**Review by oyo — CHANGES_REQUESTED** [review]
please resolve merge conflicts

**Review by oyo — APPROVED** [review]

**Pull request merged**